### ISSUE TITLE
Weights are outside of fp16 range - Re-training SSD-Mobilenet

### ISSUE THREAD

**Issue by LorenRd** [dusty-nv/jetson-inference#837]
Hi,

I'm following the guide for [Re-training SSD-Mobilenet](https://github.com/dusty-nv/jetson-inference/blob/master/docs/pytorch-ssd.md) and when I'm running the model with detectnet I've got this error:

```
[TRT]    ../builder/cudnnBuilderWeightConverters.cpp (541) - Misc Error in operator(): 1 (Weights are outside of fp16 range. A possible fix is to retrain the model with regularization to bring the magnitude of the weights down.)
[TRT]    Builder timing cache: created 65 entries, 20 hit(s)
[TRT]    ../builder/cudnnBuilderWeightConverters.cpp (541) - Misc Error in operator(): 1 (Weights are outside of fp16 range. A possible fix is to retrain the model with regularization to bring the magnitude of the weights down.)
[TRT]    device GPU, failed to build CUDA engine
[TRT]    device GPU, failed to load models/mask/ssd-mobilenet.onnx
[TRT]    detectNet -- failed to initialize.
detectnet:  failed to load detectNet model
```

I trainned the model with +800 images and 3 classes with for 30 epochs and the last .pth is:
**mb1-ssd-Epoch-29-Loss-12491.12704326923.pth**

Seems that the errors are a bit high but I wanted to try it anyway, Should I retrain it?
